clients/go/pkg/audit/interceptor.go

```diff
@@ -230,9 +235,11 @@ func (i *Interceptor) StreamInterceptor(srv interface{}, ss grpc.ServerStream, i
 	// Autofill `Payload.AuthenticationInfo.PrincipalEmail`.
 	principal, err := i.sc.RequestPrincipal(ctx)
 	if err != nil {
-		return i.handleReturnStream(ctx, ss, handler, status.Errorf(codes.FailedPrecondition,
-			"audit interceptor failed to get request principal; this is likely a result of misconfiguration of audit client (security_context): %v %v",
-			zap.Any("security_context", i.sc), zap.Error(err)))
+		logger.ErrorContext(ctx, "audit interceptor failed to get request principal",
+			"security_context", i.sc,
+			"error", err)
+		serr := status.Errorf(codes.FailedPrecondition, "audit interceptor failed to get request principal")
+		return i.handleReturnStream(ctx, ss, handler, serr)
```

Comment:
⚠️ Behavior change - returned error no longer includes information. The information wasn't useful (GoString of a pointer value).